net_connection/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "holochain_net_connection"
+version = "0.1.0"
+authors = ["neonphog <neonphog@gmail.com>"]
+
+[dependencies]
+byteorder = "1"
+failure = "0.1.3"
+holochain_core_types = { path = "../core_types" }
+rmp = "0.8"
+rmp-serde = "0.13.7"
+serde_bytes = "0.10.4"
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = { version = "1.0", features = ["preserve_order"] }
```

Comment:
i just set this to preserve order

Reply:
:+1: - awesome, yep, definitely needed : )